fdbserver/tester.actor.cpp

```diff
@@ -1838,7 +1839,19 @@ ACTOR Future<Void> runConsistencyCheckerUrgentCore(Reference<AsyncVar<Optional<C
 			// Step 2: Get testers
 			ts.clear();
 			if (!testers.present()) { // In real clusters
-				wait(store(ts, getTesters(cc, minTestersExpected)));
+				try {
+					wait(store(ts, getTesters(cc, minTestersExpected)));
+					whenFailedToGetTesterStart.reset();
+				} catch (Error& e) {
+					if (e.code() == error_code_timed_out) {
+						if (!whenFailedToGetTesterStart.present()) {
+							whenFailedToGetTesterStart = now();
+						} else if (now() - whenFailedToGetTesterStart.get() > 3600 * 24) { // 1 day
+							TraceEvent(SevError, "TesterRecruitmentTimeout").log();
```

Comment:
I am not quite sure about this, but to my understanding the `log` might not be necessary in most of the cases, since `TraceEvent` is a subclass of `BaseTraceEvent` while the destructor will ensure `log` being called. The language itself guarantees the destructor being called when the temporary r-value get destroyed. Again, I am curious why people still call `.log()`, since I have no idea while existing code has this kind of calls.